packages/core/src/routes/application.ts

```diff
@@ -1,6 +1,6 @@
 import { generateStandardId, buildIdGenerator } from '@logto/core-kit';
-import { Applications } from '@logto/schemas';
-import { object, string } from 'zod';
+import { adminConsoleAdminRoleId, Applications } from '@logto/schemas';
+import { object, string, z } from 'zod';
```

Comment:
nit: no need for object and string if we have the z imported? Prefer keep the usage consistant